java/custom/org/mozilla/Test.java

```diff
@@ -0,0 +1,6 @@
+package org.mozilla;
```

Comment:
(FWIW, in the long run, I think we should make this more explicit, removing the test-centric defaults from main.js.)